Gemfile

```diff
@@ -61,6 +61,8 @@ end
 # @see https://github.com/DataDog/dogstatsd-ruby/issues/182
 gem 'dogstatsd-ruby', '>= 3.3.0', '!= 5.0.0', '!= 5.0.1', '!= 5.1.0'
 gem 'opentracing', '>= 0.4.1'
+# TODO: Promote this to ddtrace.gemspec. (Or contrib package? Only used by contrib.)
```

Comment:
> In my opinion this kind of "small base building block" functionality could live in a separate package without much effort.

Can you clarify what you mean? In order to use this gem we need to add it to the `gemspec`, otherwise user applications won't import it and the integration won't work, right?